src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/altar/AncientPedestal.java

```diff
@@ -1,20 +1,28 @@
 package io.github.thebusybiscuit.slimefun4.implementation.items.altar;
 
+import java.util.List;
 import java.util.Optional;
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import javax.annotation.ParametersAreNonnullByDefault;
 
+import io.github.bakedlibs.dough.blocks.BlockPosition;
+import io.github.thebusybiscuit.slimefun4.core.attributes.HologramOwner;
+import io.github.thebusybiscuit.slimefun4.core.services.holograms.HologramsService;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
 import org.bukkit.GameMode;
 import org.bukkit.Location;
+import org.bukkit.NamespacedKey;
 import org.bukkit.Sound;
 import org.bukkit.block.Block;
-import org.bukkit.entity.Entity;
-import org.bukkit.entity.Item;
-import org.bukkit.entity.Player;
+import org.bukkit.entity.*;
```

Comment:
Never star import

Reply:
Idk how I missed that will fix ASAP

Reply:
Will also cleanup various imports I had from previous methods that have since been deleted

Reply:
Ah intellij did it automatically